plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/panels/managers/BuildProgressSplitterPanelManager.kt

```diff
@@ -208,19 +209,19 @@ class BuildProgressSplitterPanelManager(private val project: Project) :
             }
         }
         val updatedStatuses = when (newState) {
-            TransformationStatus.CREATED, TransformationStatus.ACCEPTED -> {
```

Comment:
`TransformationStatus.CREATED` never should have been here; our message here tells users that the transformation "has been accepted" so we should wait until the status is `ACCEPTED`

Reply:
I think during reinvent this was a concious thing to minimize the amount of states we display, the UX did not match the backend states so this did not match exactly.